api/envoy/config/core/v3/api_version.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package envoy.config.core.v3;
+
+import "envoy/type/v3/semantic_version.proto";
+
+import "udpa/annotations/status.proto";
+
+option java_package = "io.envoyproxy.envoy.config.core.v3";
+option java_outer_classname = "ApiVersionProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: API Versioning]
+
+// Describes a SemVer (<Major>.<Minor>.<Patch>) version of the API that the
+// client supports. Note that the major version is set by the proto package path, where
+// the minor and patch versions are updated upon incremental changes to the API.
+// [#not-implemented-hide:]
+message ApiVersionNumber {
```

Comment:
I haven't found a way to create `ApiVersionNumber` an alias of `SemanticVersion`, which I think would be the best way forward.
If you think that the API version will not be extended, then `SemanticVersion` can be used directly.

Reply:
The only extension I can imagine is if ever had a notion of API-wide "alpha" etc. But I strongly doubt it.